stores/postStore/actions.js

```diff
@@ -43,21 +43,29 @@ export function setLoadingPosts(isLoading) {
     isLoading,
   };
 }
+export function setItemsLeft(itemsLeft) {
+  return {
+    type: 'SET_ITEMS_LEFT',
+    itemsLeft,
+  };
+}
 export function setExpandedPost(postId) {
   return {
     type: 'SET_EXPANDED_POST',
     postId,
   };
 }
 
-export function requestPosts(postId) {
-  return { type: 'REQUEST_POSTS', postId };
+export function requestPosts(postId, pageNumber) {
+  return { type: 'REQUEST_POSTS', postId, pageNumber };
 }
 
-export function receivePosts(posts) {
+export function receivePosts(posts, pageNumber, itemsLeft) {
   return {
     type: 'RECEIVE_POSTS',
     posts,
+    pageNumber,
+    itemsLeft,
```

Comment:
Since you made `setItemsLeft` action creator, you don't need to pass `itemsLeft` here.